File: OlaMundo/app/src/main/java/com/example/olamundo/MainActivity.java
package com.example.olamundo;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
public class MainActivity extends AppCompatActivity {

    TextView msg;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        msg = (TextView) findViewById(R.id.msgView);
    }

    public void clicou(View view) {
        msg.setText("Ola, Mundo");
    }
}

File: xml/Kotlin/Gerador De Senhas/app/src/main/java/com/example/geradordesenhas/MainActivity.kt
package com.example.geradordesenhas

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.geradordesenhas.databinding.ActivityMainBinding
import kotlin.random.Random

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    private val alfabeto = arrayOf("A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z")

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btGerarSenha.setOnClickListener {
            val letra1 = alfabeto.random()
            val letra2 = alfabeto.random()
            val numero = Random.nextInt(1000,9999)
            binding.tvSenha.text = "$letra1$letra2-$numero"
        }
    }
}